### PR TITLE
GH-8596 Don't do configuration updates if cart updates are pending

### LINKED ISSUE #8596: Click add to cart and then make quick changes to the configuration with slow network.

### PR DESCRIPTION
**Pull request by ChristophHi**

### CODE REVIEW

File: projects/core/src/configurator/commons/facade/configurator-cart.service.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export class ConfiguratorCartService {
    * Checks if there are no updates for the active cart pending
```
Comment (schuppell) [inline projects/core/src/configurator/commons/facade/configurator-cart.service.ts:26]: This description is not quite correct.
Change to: Checks if there are updates for the active cart pending.

File: projects/core/src/configurator/commons/facade/configurator-cart.service.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ export class ConfiguratorCartService {
...
    * @returns Observable emitting that no changes are pending
```
Comment (schuppell) [inline projects/core/src/configurator/commons/facade/configurator-cart.service.ts:27]: This description is not correct
Change to: Observable emitting weather changes are pending or not

**Review by schuppell — CHANGES_REQUESTED** [review]
See Comments regarding JS Doc

File: projects/core/src/configurator/commons/facade/configurator-cart.service.ts (hunk 1 above)
Comment (ChristophHi) [reply projects/core/src/configurator/commons/facade/configurator-cart.service.ts:26]: You are right. But I will delete those methods anyhow (subject of next PR on Larisas for checking), so doesn't make sense to change...

Comment (ChristophHi) [reply projects/core/src/configurator/commons/facade/configurator-cart.service.ts:26]: Method will be deleted...

File: projects/core/src/configurator/commons/facade/configurator-cart.service.ts (hunk 2 above)
Comment (ChristophHi) [reply projects/core/src/configurator/commons/facade/configurator-cart.service.ts:27]: Method will be deleted...

**Review by schuppell — APPROVED** [review]

**Pull request merged**